modules/evs/ut-coverage/evs_UT.c

```diff
@@ -1814,6 +1862,126 @@ void Test_InvalidCmd(void)
+                UtAssert_UINT32_EQ(EVS_Retval, CFE_EVS_APP_SQUELCHED);
+            }
+            else
+            {
+                UtAssert_UINT32_EQ(SnapshotData.Count, CFE_PLATFORM_EVS_MAX_APP_EVENT_BURST + 1);
+                UtAssert_UINT32_EQ(EVS_Retval, CFE_EVS_APP_SQUELCHED);
+            }
+        }
+        /*
+         * Test unsquelching w/ passage of time
+         */
+        /* Allow time for tokens to return */
+        InjectedTime = OS_TimeAssembleFromMilliseconds(0, 1000 * CFE_PLATFORM_EVS_MAX_APP_EVENT_BURST /
+                                                                  CFE_PLATFORM_EVS_APP_EVENTS_PER_SEC +
+                                                              1000 / CFE_PLATFORM_EVS_APP_EVENTS_PER_SEC);
+        UT_SetDataBuffer(UT_KEY(CFE_PSP_GetTime), &InjectedTime, sizeof(InjectedTime), false);
+        EVS_Retval = SendEventFuncs[j](EVENT_ID);
+        UtAssert_UINT32_EQ(SnapshotData.Count, CFE_PLATFORM_EVS_MAX_APP_EVENT_BURST + 2);
+        UtAssert_UINT32_EQ(CapturedTlm.Payload.PacketID.EventID, EVENT_ID);
+        UtAssert_UINT32_EQ(EVS_Retval, CFE_SUCCESS);
+
+        /*
+         * Test re-squelching event message
+         */
+        UT_SetDataBuffer(UT_KEY(CFE_PSP_GetTime), &InjectedTime, sizeof(InjectedTime), false);
+        EVS_Retval = SendEventFuncs[j](EVENT_ID);
+        UtAssert_UINT32_EQ(SnapshotData.Count, CFE_PLATFORM_EVS_MAX_APP_EVENT_BURST + 3);
+        UtAssert_UINT32_EQ(CapturedTlm.Payload.PacketID.EventID, CFE_EVS_SQUELCHED_ERR_EID);
+        UtAssert_UINT32_EQ(EVS_Retval, CFE_EVS_APP_SQUELCHED);
+
+        /*
+         * Test re-squelching
+         */
+        UT_SetDataBuffer(UT_KEY(CFE_PSP_GetTime), &InjectedTime, sizeof(InjectedTime), false);
+        EVS_Retval = SendEventFuncs[j](EVENT_ID);
+        UtAssert_UINT32_EQ(SnapshotData.Count, CFE_PLATFORM_EVS_MAX_APP_EVENT_BURST + 3);
+        UtAssert_UINT32_EQ(EVS_Retval, CFE_EVS_APP_SQUELCHED);
+
+        /*
+         * Test positive saturation of tokens
+         */
+        InjectedTime = OS_TimeAssembleFromMilliseconds(0, 3 * 1000 * CFE_PLATFORM_EVS_MAX_APP_EVENT_BURST /
+                                                                  CFE_PLATFORM_EVS_APP_EVENTS_PER_SEC +
+                                                              1000 / CFE_PLATFORM_EVS_APP_EVENTS_PER_SEC);
+        UT_SetDataBuffer(UT_KEY(CFE_PSP_GetTime), &InjectedTime, sizeof(InjectedTime), false);
+        EVS_Retval = SendEventFuncs[j](EVENT_ID);
+        UtAssert_UINT32_EQ(SnapshotData.Count, CFE_PLATFORM_EVS_MAX_APP_EVENT_BURST + 4);
+        UtAssert_UINT32_EQ(CapturedTlm.Payload.PacketID.EventID, EVENT_ID);
+        UtAssert_UINT32_EQ(EVS_Retval, CFE_SUCCESS);
+        UtAssert_UINT32_EQ(AppDataPtr->SquelchedCount, CFE_EVS_MAX_SQUELCH_COUNT);
+
+        /*
+         * Test boundary condition where SquelchTokens > -EVENT_COST && CreditCount >= EVENT_COST
```

Comment:
Update comment to state `SquelchTokens > |-EVENT_COST| && CreditCount >= EVENT_COST && CreditCount < |SquelchTokens|`

Reply:
This test was added to address branch coverage of the commented case in the if statement, specifically where CreditCount >= EVENT_COST.  The intention was to not cross the threshold or cause SendSquelchEvent = true.